draft-packages/hero-card/KaizenDraft/HeroCard/HeroCard.scss

```diff
@@ -39,6 +38,26 @@ $left-content__width: 221px;
     width: 33%;
     max-width: 450px;
   }
+
+  &.wisteria700 {
+    background-color: $kz-color-wisteria-700;
+  }
+
+  &.wisteria200 {
+    background-color: $kz-color-wisteria-200;
+  }
+
+  &.cluny200 {
+    background-color: $kz-color-cluny-200;
+  }
+
+  &.seedling200 {
+    background-color: $kz-color-seedling-200;
+  }
+
+  &.transparent {
+    background-color: transparent;
+  }
```

Comment:
This one seems to not be connected?

Reply:
good catch, this is a hold over from my trial with transparency, I'll remove.